go.mod

```diff
@@ -3,54 +3,47 @@ module github.com/ethereum/hive
 go 1.12
 
 require (
+	github.com/Microsoft/hcsshim v0.8.10 // indirect
+	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
 	github.com/allegro/bigcache v1.2.1 // indirect
-	github.com/apilayer/freegeoip v3.5.0+incompatible // indirect
-	github.com/aristanetworks/goarista v0.0.0-20190712234253-ed1100a1c015 // indirect
-	github.com/btcsuite/btcd v0.20.1-beta // indirect
+	github.com/aristanetworks/goarista v0.0.0-20201012165903-2cb20defcd66 // indirect
+	github.com/btcsuite/btcd v0.21.0-beta // indirect
 	github.com/cespare/cp v1.1.1 // indirect
+	github.com/containerd/containerd v1.4.1 // indirect
+	github.com/containerd/continuity v0.0.0-20200928162600-f2cc35102c2a // indirect
 	github.com/deckarep/golang-set v1.7.1 // indirect
 	github.com/edsrzf/mmap-go v1.0.0 // indirect
-	github.com/elastic/gosigar v0.10.4 // indirect
-	github.com/ethereum/go-ethereum v1.9.1
+	github.com/ethereum/go-ethereum v1.9.23
```

Comment:
It seems that you somehow upgraded all the dependencies here. Please revert the go.mod changes.

Reply:
ah yeah i ran a go mod tidy